ForceDependentKinematics/Downloads/DemoSimpleKnee4.any

```diff
@@ -33,15 +34,26 @@ Main = {
               RGB = {0, 0, 1};
             };
           };
+        };  
+        
+        // Define a STL file as femoral condyle for the joint geometry. The tibial 
```

Comment:
Downloading `DemoSimpleKnee4.any` does not include the femoral stl file - could we bundle those two perhaps?

Reply:
I will add that in a separate PR.